recipes/grpc-cpp/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "grpc-cpp" %}
+{% set version = "1.18.0" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/grpc/grpc/archive/v{{ version }}.tar.gz
+    sha256: 069a52a166382dd7b99bf8e7e805f6af40d797cfcee5f80e530ca3fc75fd06e2
+    patches:
+      # Building the Objective C plugin fails on Windows, skip it.
+      - 0001-remove-objective-c-target.patch  # [win]
+
+build:
+  number: 1000
+  # Requires C++ 11, VS 2008 is not supported
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    # `protoc` is also used for building
+    - libprotobuf 3.6.1
```

Comment:
Is libprotobuf really needed on the build system?  If you are linking to it, it should just be needed in host

Reply:
`protoc` is required to compile protobuf definitions to sources.

Reply:
+1